### PR TITLE
small logging fixes

### PR DESCRIPTION
**Pull request by tangiel**

### CODE REVIEW

File: endpoints-control/src/main/java/com/google/api/control/Client.java [hunk 1]
```diff
@@ -56,6 +56,10 @@
 public class Client {
   private static final Logger log = Logger.getLogger(Client.class.getName());
   private static final String CLIENT_APPLICATION_NAME = "Service Control Client";
+  private static final String BACKGROUND_THREAD_ERROR =
+      "The scheduler thread was unable to start. This means that metric reporting will only be "
+      + "done periodically after requests are served. If your API is low traffic (below 1 QPS), "
```
Comment (bradfriedman) [inline endpoints-control/src/main/java/com/google/api/control/Client.java]: nit: low-traffic

Comment (bradfriedman) [inline endpoints-control/src/main/java/com/google/api/control/Client.java]: Maybe change QPS to 'query per second' just to be less jargon-y

Comment (tangiel) [inline endpoints-control/src/main/java/com/google/api/control/Client.java]: Done

Comment (tangiel) [inline endpoints-control/src/main/java/com/google/api/control/Client.java]: Done

**Pull request merged**